apis/apps/v1alpha1/backuppolicytemplate_types.go

```diff
@@ -25,12 +25,12 @@ import (
 // BackupPolicyTemplateSpec contains the settings in a BackupPolicyTemplate.
 type BackupPolicyTemplateSpec struct {
 	// Specifies the name of a ClusterDefinition.
-	//
 	// This is an immutable attribute that cannot be changed after creation.
+	// And this field is deprecated since v0.9.
```

Comment:
Why?

Reply:
backupPolicyTemplate needs to find the corresponding clusterComponentDefinition in clusterDefinition before 0.9. but we instead of `ComponentDefinition` API since 0.9.

Reply:
I see, should also add a kubebuilder marker: 

// +kubebuilder:deprecatedversion:warning="This field has been deprecated since 0.9.0, consider using the ComponentDef instead"